src/javascript/app/pages/loader.jsx

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable import/no-unresolved */
+import React, { useEffect, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { Skeleton } from '@deriv-com/quill-ui';
+import Portal from './portal';
+import { getElementById } from '../../_common/common_functions';
+import { useContractChange } from '../hooks/events';
+import dataManager from '../common/data_manager';
+
+const PurchaseSectionLoader = ({ hideBottom }) => (
+    <>
+        <Skeleton.Square rounded fullWidth height={30} />
+        <Skeleton.Square rounded fullWidth height={130} />
+        {!hideBottom && (
+            <>
+                <Skeleton.Square rounded fullWidth height={30} />
+                <Skeleton.Square rounded fullWidth height={130} />
+            </>
+        )}
+    </>
+);
```

Comment:
PurchaseSectionLoader is quite small and used just once. Are you sure it's worth being separated? :)

Reply:
`PurchaseSectionLoader` will be used in the `Purchase Section` we just didn't get a time to incorporate it due to high priority bugs.